### ISSUE TITLE
add ability to pre configure admin user

### ISSUE THREAD

**Issue by nabulaer** [sunggun-yu/chef-mongodb3#33]
when enabling authentication , can not login without a user, so allow config from chef the default admin user and password much like mongodb cookbook allow.

**Comment by LordBrain** [comment]
Yes, I am having this same problem. Has anyone found a work around for it? It is creating a major blocker for us.

**Comment by NickvdMeij** [comment]
+1

**Comment by amsdard** [comment]
+1

**Comment by amsdard** [comment]
Hello everyone, I've added pull request with this very feature. Please take a look, test and let me know if there is anything to improve.
